src/deploy/NVA_build/builder.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.centos.org/centos:8 
+FROM centos:8 
```

Comment:
@jeniawhite Is this change intended? CC @liranmauda

Reply:
My mistake, the repo did not work for me. Disregard.
Thank you.